install_config/master_node_configuration.adoc

```diff
@@ -1299,6 +1299,105 @@ For more information on advanced audit, see the
 link:https://kubernetes.io/docs/tasks/debug-application-cluster/audit[Kubernetes
 documentation]
 
+[[master-config-tls-cipher]]
+=== Specifying TLS ciphers for etcd
+
+You can specify the xref:../architecture/index.adoc#arch-index-how-is-it-secured-tls[supported
+TLS ciphers] to use in communication between the master and etcd servers.
+
+. On each etcd node, upgrade etcd:
++
+----
+# yum update etcd iptables-services
+----
+
+. Confirm that your etcd version is 3.2.22 or later:
++
+----
+# etcd --version
+etcd Version: 3.2.22
+----
+
+. On each master host, specify the ciphers to enable in the
+`/etc/origin/master/master-config.yaml` file:
++
+----
+servingInfo:
+  ...
```

Comment:
it seems miss 'minTLSVersion: VersionTLS12', right?

Reply:
@geliu2016 
1) Have I placed 'minTLSVersion: VersionTLS12' correctly in the block?
2) Is this going to work in 3.9 and 3.7? I know the master restart commands will be different for the different versions, but is this approach otherwise valid for both of them?

Reply:
1. Right, I got this msg from dev in bug's comment, you could double check the bug
2. I try it on 3.9 and it work, and master restart commands should vaild for this options
3. Right

Reply:
1. Right. The code in the dev bug looked like this, so I assume that the new line should go under `servingInfo:` 
```
  minTLSVersion: VersionTLS12 

  cipherSuites: 
  - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 
  - TLS_RSA_WITH_AES_256_CBC_SHA 
  - TLS_RSA_WITH_AES_128_CBC_SHA
```
2. Good.
3. I'll put that command in the 3.9 and 3.7 PRs.